web/.eslintrc.cjs

```diff
@@ -32,6 +32,7 @@ module.exports = {
 				caughtErrorsIgnorePattern: "^_",
 			},
 		],
+		"solid/prefer-for": "off",
```

Comment:
This should be at the top of the relevant file (as a comment) instead. Putting it here disables the lint for all files which is not what we want, because it's quite useful when using SolidJS proper.